### PR TITLE
Cp post ga fixes

### PR DESCRIPTION
**Pull request by moxiegirl**
Cherry picking the fixes that came in from the guys.

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
@nishanttotla @dongluochen can you double check this cherry pick for me?

**Comment by dongluochen** [comment]
LGTM

File: docs/install-w-machine.md [hunk 1]
```diff
@@ -101,20 +101,20 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
...
-        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1   
+        agent1    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.9.1
+        agent2    -        virtualbox   Running   tcp://192.168.99.103:2376           v1.9.1
+        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1
 
 
 2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary Swarm manager on `manager`.
 
-        $ docker run -d -p <your_selected_port>:2375 -t swarm manage token://<cluster_id> >
+        $ docker run -d -p <your_selected_port>:3376 -t -v /var/lib/boot2docker:/certs:ro swarm manage -H 0.0.0.0:3376 --tlsverify --tlscacert=/certs/ca.pem --tlscert=/certs/server.pem --tlskey=/certs/server-key.pm token://<cluster_id>
```
Comment (nishanttotla) [inline docs/install-w-machine.md]: I made a mistake in the `swarm` docs, will correct it there, sorry about that. Please update from `--tlskey=/certs/server-key.pm` to `--tlskey=/certs/server-key.pem`

File: docs/install-w-machine.md [hunk 2]
```diff
@@ -101,20 +101,20 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
...
 
     For example:
 
-        $ docker run -d -p 2375:2375 -t swarm manage token://0ac50ef75c9739f5bfeeaf00503d4e6e
+        $ docker run -d -p 3376:3376 -t -v /var/lib/boot2docker:/certs:ro swarm manage -H 0.0.0.0:3376 --tlsverify --tlscacert=/certs/ca.pem --tlscert=/certs/server.pem --tlskey=/certs/server-key.pm swarm manage token://0ac50ef75c9739f5bfeeaf00503d4e6e
```
Comment (nishanttotla) [inline docs/install-w-machine.md]: Here too!

File: docs/install-w-machine.md (hunk 1 above)
Comment (nishanttotla) [inline docs/install-w-machine.md]: I made a fix in #1784. Sorry about the complication, hope it's possible to pull that commit in too!

**Comment by vieux** [comment]
@moxiegirl can you please add this one https://github.com/docker/swarm/pull/1784 ?

**Comment by vieux** [comment]
LGTM

**Pull request merged**